tools/lkl/include/lkl.h

```diff
@@ -229,6 +233,17 @@ struct lkl_netdev {
 int lkl_netdev_add(struct lkl_netdev *nd, void *mac);
 
 /**
+* lkl_netdevs_remove - destroy all network devices
```

Comment:
@stfairy Do you see how this work with your facade? Exposing lkl_netdevs_remove in facade?